R/viz_plots.R

```diff
@@ -472,20 +472,23 @@ plot_riskmatrix <- function(df, region = "WHO Region", v = T, h = T) {
 #' @title plot_vaxcoverage
 #' @description Plot vaccination coverage (partial or fully) by WHO region(s), State region(s), or Income levels.
 #' @param df A dataframe with vaccination stats.
-#'
+#' @param type = "People" (default) for people who received at least 1 does, "Fully" for people who completed the initial protocol, "Booster" for booster doses only, and "Pop18" for people people vaccinated in the vaccine eligible population
+#' @param by_cat = "State Region" (default), "WHO Region" or "Income Level"
+#' @param plot = "Population" to display plot bubble size based on population size of each country, or "Vaccination" to display bubble size based on total vaccine doses administered in each country
 
 #'
 #' @export
 
-plot_vaxcoverage <- function(df, type = c("People", "Fully", "Booster", "Pop18"), by_cat = "State Region") {
+plot_vaxcoverage <- function(df, type = c("People", "Fully", "Booster", "Pop18"), by_cat = "State Region", plot = c("Population", "Vaccination")) {
```

Comment:
Can we swap the ordering of the plot argument to put vaccination first? (otherwise it'll default to population-based if plot isn't explicitly specified as "Vaccination")